gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -321,6 +319,22 @@ def _get_test_configs(include_large_tests):
   return test_configs
 
 
+def _get_test_file_path(include_large_tests, include_huge_tests):
+  if include_huge_tests:
+    test_file_path = os.path.join(
+        os.getcwd(), 'gcp_variant_transforms/testing/integration')
```

Comment:
nit: consider adding a new constant (e.g. `_BASE_TEST_FOLDER = gcp_variant_transforms/testing/integration`) and use it here and below to make the strings shorter.

Reply:
Done.